starlette/testclient.py

```diff
@@ -33,6 +33,9 @@
 ASGI3App = typing.Callable[[Scope, Receive, Send], typing.Awaitable[None]]
 
 
+_RequestData = typing.Mapping[str, typing.Union[str, typing.Iterable[str]]]
```

Comment:
probably dumb question but  cant we use httpx types directly into starlette ?

Reply:
If we're willing to bump the min version to the next unreleased version, then yes: https://github.com/encode/httpx/commit/8152c4facd0f71e0f376287e41a0810a60fec9c6

I figured I'd just fix it here since I'm doing related stuff and it's a one liner, that way we don't have to require everyone to bump up their httpx version just for a single type annotation

Reply:
I'd rather bump the req and have the types manages in a single place, but ymmv :)